examples/RESOURCES.md

```diff
@@ -0,0 +1,11 @@
+# Resources and examples
+
+> Useful tools or projects as example of main Puppeteer features.
+
+## Web scraping
+- [Puppetron](https://github.com/cheeaun/puppetron) - Non-production (Headless Chrome Node API)-based rendering solution.
+ - [Thal](https://github.com/emadehsan/thal) - Getting started with Puppeteer and Chrome Headless for Web Scraping.
```

Comment:
Let's just link to the article:
https://medium.com/@e_mad_ehsan/getting-started-with-puppeteer-and-chrome-headless-for-web-scrapping-6bf5979dee3e